stock_quant_manual_assign/wizard/assign_manual_quants.py

```diff
@@ -17,7 +17,7 @@ def check_qty(self):
             if record.quants_lines:
                 move = self.env['stock.move'].browse(
                     self.env.context['active_id'])
-                if record.lines_qty > move.product_uom_qty:
+                if record.lines_qty > move.product_qty:
```

Comment:
I use this module to assign quants in "Manufacturing Orders/Products to Consume" (MRP module).
UoM in my BoM (I use "g" because quantities are very small) is different from the default UoM of the product ("Kg" because it is the UoM I use for the inventory).
When I select a "stock.move" in "Products to consume", the UoM is the BoM one ("g") and when I open the wizard "Manual Quants", the quant UoM is the default UoM of the product ("Kg"). So there is an issue. That's the reason why I use move.product_qty which seems to be in the same UoM of the quant. In my tests, move.product_qty and move.product_uom_qty have the same value when the UoM ratio is applied.

But Anoher solution, perhaps a better one, is to convert move.product_uom_qty to the default product UoM by using move.product_uom.ratio. I saw a conversion method in UoM. Do you known if quant.qty is always in the default UoM of the product because there isn't UoM field in Quant ?